### PR TITLE
I made some change in EventsController#action

### PR DESCRIPTION
**Pull request by SuperManEver**
changed 'each' to more appropriate 'reduce'

### CODE REVIEW

**Comment by diraulo** [comment]
Hi @SuperManEver is this PR linked to a bug entry or feature on Pivotal Tracker? 
Also you have several test currently failing as a result to your changes :-)

File: Gemfile [hunk 1]
```diff
@@ -63,7 +63,7 @@ gem 'jvectormap-rails', '~> 1.0.0' #jVectorMap for the Rails asset pipeline
 group :test do
   gem 'capybara' # Simulates user actions for cucumber
   gem 'cucumber-rails', :require => false # Cucmber features
-  gem 'capybara-webkit'  # Headless driver for capybara
+#  gem 'capybara-webkit'  # Headless driver for capybara
```
Comment (diraulo) [inline Gemfile]: any reasons why you commented this out?

Comment (SuperManEver) [inline Gemfile]: I had problem with installing it and I decided to skipped it entirely

**Comment by tansaku** [comment]
hi @SuperManEver - are you blocked on this?  can we help?

**Comment by tansaku** [comment]
@SuperManEver we're going to close this for the time being - looks like you were proposing some good refactoring here - please feel free to re-open if you want to chat with us about it and get it merged in

**Pull request closed without merging**